### PR TITLE
Release v0.22.0

### PR DESCRIPTION
**Pull request by alexvanin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,44 @@
...
+- Tagging and ACL notifications (#361) 
+- AWSv4 signer package to improve compatibility with S3 clients (#528)
+- Extension mimetype detector (#289)
+- Default params documentation (#592)
+- Health metric (#600)
+- Parallel object listing (#525)
+- Tree service (see commit links from #609)
+
+### Changed
+- Cache empty bucket settings (#439)
```
Comment (masterSplinter01) [inline CHANGELOG.md:33]: Should we mention this change? As for me, it seems like internal change which doesn’t affect anything (for users)

Comment (alexvanin) [reply CHANGELOG.md:33]: Why not. Tree service is also an internal change, however it changes many things.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,44 @@
...
+- Reduce number of network requests (#439, #441)
+- Reduce number of object.Head request during object upload (#)
+- Authmate renamed to s3-authmate (#518)
```
Comment (masterSplinter01) [inline CHANGELOG.md:36]: I think the sentence should be reworded as passive voice or ‘Renamed automate to s3-authmate’?

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,44 @@
...
 
+## [0.22.0] - 2022-07-25
+
+Tree service support
+
+### Fixed
+- Error logging (#450)
+- Default bucket location constraint (#463)
+- Suspended versioning status (#462)
+- CodeQL warnings (#489, #539)
```
Comment (KirillovDenis) [inline CHANGELOG.md:13]: Should mention https://github.com/nspcc-dev/neofs-s3-gw/issues/522 too I suppose

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,44 @@
...
+- Extension mimetype detector (#289)
+- Default params documentation (#592)
+- Health metric (#600)
+- Parallel object listing (#525)
+- Tree service (see commit links from #609)
+
+### Changed
+- Cache empty bucket settings (#439)
+- Reduce number of network requests (#439, #441)
+- Reduce number of object.Head request during object upload (#)
```
Comment (KirillovDenis) [inline CHANGELOG.md:35]: Missed link? Or we wait for something?

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,43 @@
...
+- Health metric (#600)
+- Parallel object listing (#525)
+- Tree service (see commit links from #609)
+
+### Changed
+- Cache empty bucket settings (#439)
+- Reduce number of network requests (#439, #441)
+- Renamed authmate to s3-authmate (#518)
+- Version output (#578)
+- Improved error messaged (#539)
```
Comment (masterSplinter01) [inline CHANGELOG.md:37]: Messages?

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,43 @@
...
+- AWSv4 signer package to improve compatibility with S3 clients (#528)
+- Extension mimetype detector (#289)
+- Default params documentation (#592)
+- Health metric (#600)
+- Parallel object listing (#525)
+- Tree service (see commit links from #609)
+
+### Changed
+- Cache empty bucket settings (#439)
+- Reduce number of network requests (#439, #441)
```
Comment (KirillovDenis) [inline CHANGELOG.md:34]: Is it ok to have two point to refer to the same issue? (#439) 
Can we keep only last one `Reduce number of network requests (#439, #441)`?

File: CHANGELOG.md (hunk 5 above)
Comment (alexvanin) [reply CHANGELOG.md]: Spellchecker is not so good as I want it to be :disappointed:

File: CHANGELOG.md (hunk 6 above)
Comment (alexvanin) [reply CHANGELOG.md:34]: Yep, the same as https://github.com/nspcc-dev/neofs-s3-gw/pull/621#discussion_r928931743

**Review by KirillovDenis — APPROVED** [review]

**Review by masterSplinter01 — APPROVED** [review]

**Pull request merged**